src/operator_ceed.c

```diff
@@ -5,6 +5,7 @@
 #include <private/rdysedimentimpl.h>
 #include <private/rdysweimpl.h>
 
+#include "petscsystypes.h"
```

Comment:
Is this line necessary or inserted by your editor? Since it's not at a relative path, if it's needed, it should be
```suggestion
#include <petscsystypes.h>
```